packages/validator/src/commands/validateCmd.ts

```diff
@@ -0,0 +1,24 @@
+import * as utils from './utils';
+import { validate } from '../validator';
+import { Log, templates } from '../types';
+
+let template = process.env.npm_config_template || process.argv[2];
```

Comment:
Using `let` and reassigning variables is usually not a good practice, especially when the content is different (like here). Code is hard to follow that way.

Reply:
I would usually agree, however in this case, there is very few uses of the variable and it might be even harder to follow when rewritten using multiple variables.